WordPress/src/main/java/org/wordpress/android/ui/voicetocontent/VoiceToContentUseCase.kt

```diff
@@ -25,6 +30,10 @@ class VoiceToContentUseCase @Inject constructor(
         file: File
     ): VoiceToContentResult =
         withContext(Dispatchers.IO) {
+            if (!networkUtilsWrapper.isNetworkAvailable()) {
```

Comment:
Is it common on Android to stop the request without sending it in case the app thinks network is unavailable. On iOS, it's considered a bad practice because the respective API usually lags behind the real status a bit and is not always accurate.

Reply:
@kean - yes, this is the way it's done on Android. Something perhaps for the future app team to align?